frontend/src/translations/ua/common.json

```
@@ -0,0 +1,566 @@
+      },
+      "ValidatorRow": {
+        "warning_tip": "Валідатори 1 - ${node_tip_max} мають кумулятивну частку стейкінгу понад 33%. Делегування валідаторам нижче покращує децентралізацію мережі."
+      },
+      "ValidatorMainRow": {
+        "state": {
+          "active": {
+            "text": "поточні ноди валідаторів",
+            "title": "Активні"
+          },
+          "joining": {
+            "title": "Приєднання",
+            "text": "ноди валідаторів, які будуть в наступній епосі"
+          },
+          "leaving": {
+            "title": "Видалення",
+            "text": "ноди валідаторів, які будуть видалені в наступній епосі"
+          },
+          "proposal": {
+            "title": "Пропозиція",
+            "text": "застейкані ноди, які будуть новими нодами валідаторів"
+          },
+          "idle": {
+            "title": "Неактивні",
+            "text": "пул стейкінгу, який має менше 20% поточної ціни місця валідатора"
+          },
+          "newcomer": {
+            "title": "Новачки",
+            "text": "пул стейкінгу, який має мінімум 20% поточної ціни місця валідатора"
+          },
+          "onHold": {
+            "title": "На утриманні",
+            "text": "пул стейкінгу має достатньо токенів для валідування, але ще не подав пропозиції щодо стейкінгу"
+          }
+        }
+      },
+      "ValidatorTelemetryRow": {
+        "uptime": {
+          "text": "Час безперервної роботи оцінюється відношенням кількості створених блоків до кількості очікуваних блоків.",
+          "title": "Час безперервної роботи"
+        },
+        "latest_produced_block": {
+          "text": "Висота блоку, про яку повідомляє нод перевірки в останньому пульсі телеметрії.",
+          "title": "Поточний блок"
+        },
+        "latest_telemetry_update": {
+          "text": "Телеметрія - це регулярне сповіщення, яке надходить від вузлів, та яке включає загальну інформацію, як-от останню відому висоту блоку та версію агента NEAR Protocol (nearcore).",
+          "title": "Останнє оновлення телеметрії"
+        },
+        "node_agent_name": {
+          "text": "Протокол NEAR може мати кілька реалізацій, тому агент - це ім'я цієї реалізації, де 'near-rs' - це <nearCoreLink>офіційна реалізація</nearCoreLink>..",
+          "title": "Ім'я агента ноду"
+        },
+        "node_agent_version_or_build": {
+          "title": " Версія агента ноди/ Білд "
+        },
+        "produced_blocks_and_chunks": "зроблено ${num_produced_blocks}/${num_expected_blocks} блоків, ${num_produced_chunks}/${num_expected_chunks} чанків"
+      },
+      "ValidatorMetadataRow": {
+        "pool_info": {
+          "website": "Web",
+          "email": "Email",
+          "twitter": "Twitter",
+          "discord": "Discord",
+          "description": "Опис"
+        },
+        "pool_info_tip": "Якщо ви є власником нода, не соромтеся заповнювати всі <poolLink>дані</poolLink> для просування власного нода!!"
```

```suggestion
        "pool_info_tip": "Якщо ви є власником ноди, не соромтеся заповнювати всі <poolLink>дані</poolLink> для просування власної ноди!"
```

We can think about better name instead of "нода"

---

"Вузол"? Зазвичай node в мережевій термінології перекладається як вузли